### ISSUE TITLE
How to get the find domNode's innerHTML ?

### ISSUE THREAD

**Issue by BiteBit** [rc0x03/node-osmosis#26]
```
osmosis = require 'osmosis'

osmosis
.get('http://www.cnbeta.com/')
.find('div.hd > div.title > a')
.follow('@href')
.set({
    'content': 'how to get domNode's innerHTML'
})
.data (listing)->
  console.log listing
```

default is get text

**Comment by milescui** [comment]
I want to know, too

**Comment by rc0x03** [comment]
Osmosis 0.0.7 will support `:html` and  `:source` in selector strings. For now you can use `.then` and `context.toString()`. Thank you for being patient.

**Comment by rc0x03** [comment]
Fixed in Osmosis 0.0.7
